supabase/migrations/20240924172134_add_hypercert_metadata_for_hyperboards.sql

```diff
@@ -0,0 +1,386 @@
+
+alter table "public"."blueprints"
+    drop column "admin_id";
+
+CREATE UNIQUE INDEX blueprint_admins_pkey ON public.blueprint_admins USING btree (user_id, blueprint_id);
+
+alter table "public"."blueprint_admins"
+    add constraint "blueprint_admins_pkey" PRIMARY KEY using index "blueprint_admins_pkey";
+
+alter table "public"."blueprint_admins"
+    add constraint "blueprint_admins_blueprint_id_fkey" FOREIGN KEY (blueprint_id) REFERENCES blueprints (id) ON UPDATE CASCADE ON DELETE CASCADE not valid;
+
+alter table "public"."blueprint_admins"
+    validate constraint "blueprint_admins_blueprint_id_fkey";
+
+alter table "public"."blueprint_admins"
+    add constraint "blueprint_admins_user_id_fkey" FOREIGN KEY (user_id) REFERENCES users (id) ON UPDATE CASCADE ON DELETE CASCADE not valid;
+
+alter table "public"."blueprint_admins"
+    validate constraint "blueprint_admins_user_id_fkey";
+
+grant delete on table "public"."blueprint_admins" to "anon";
+
+grant insert on table "public"."blueprint_admins" to "anon";
+
+grant references on table "public"."blueprint_admins" to "anon";
+
+grant select on table "public"."blueprint_admins" to "anon";
+
+grant trigger on table "public"."blueprint_admins" to "anon";
+
+grant truncate on table "public"."blueprint_admins" to "anon";
+
+grant update on table "public"."blueprint_admins" to "anon";
+
+grant delete on table "public"."blueprint_admins" to "authenticated";
+
+grant insert on table "public"."blueprint_admins" to "authenticated";
+
+grant references on table "public"."blueprint_admins" to "authenticated";
+
+grant select on table "public"."blueprint_admins" to "authenticated";
+
+grant trigger on table "public"."blueprint_admins" to "authenticated";
+
+grant truncate on table "public"."blueprint_admins" to "authenticated";
+
+grant update on table "public"."blueprint_admins" to "authenticated";
+
+grant delete on table "public"."blueprint_admins" to "service_role";
+
+grant insert on table "public"."blueprint_admins" to "service_role";
+
+grant references on table "public"."blueprint_admins" to "service_role";
+
+grant select on table "public"."blueprint_admins" to "service_role";
+
+grant trigger on table "public"."blueprint_admins" to "service_role";
+
+grant truncate on table "public"."blueprint_admins" to "service_role";
+
+grant update on table "public"."blueprint_admins" to "service_role";
+
+alter table "public"."collections"
+    drop column "admin_id";
+
+alter table "public"."hypercerts"
+    drop column "admin_id";
+
+alter table "public"."hypercerts"
+    drop column "chain_id";
+
+alter table "public"."blueprints"
+    drop column "display_size";
+
+alter table "public"."collection_blueprints"
+    add column "display_size" bigint not null;
+
+alter table "public"."hyperboards"
+    drop column "admin_id";
+
+alter table "public"."collections"
+    drop column "chain_id";
+
+alter table "public"."collections"
+    add column "chain_ids" bigint[] not null;
```

Comment:
we use `numeric(78,0)`, see other bigint values in our databases

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/commit/9436dd6af93162b30d97951db3cc68888e364666

Reply:
Thanks, but wasn't display size a number? You mentioned values like 1, 2, etc? The reason for `numeric(78,0)` is because the coilumn represents an on-chain value declared as uint256, which doesn't fit in a bigint

Reply:
Worked either way, but changed it back for consistency.